src/content/docs/apm/agents/net-agent/net-agent-api/net-agent-api.mdx

```diff
@@ -667,38 +636,47 @@ The following list contains the different calls you can make with the API, inclu
         ### Usage considerations
 
         * [Distributed tracing must be enabled](/docs/agents/net-agent/configuration/net-agent-configuration#distributed_tracing).
-        * `AcceptDistributedTraceHeaders` will be ignored if `InsertDistributedTraceHeaders` or `AcceptDistributedTraceHeaders` has already been called for this transaction.
+        * This API can only be used within the context of an existing [transaction](/docs/apm/agents/net-agent/custom-instrumentation/introduction-net-custom-instrumentation/#new-existing).
 
         ### Example
 
+        You can find a complete example that you can build and run to demonstrate the use of this API [here](https://github.com/newrelic/newrelic-dotnet-examples/tree/main/custom-distributed-tracing).
+
         ```cs
-        HttpContext httpContext = HttpContext.Current;
+        // Get a reference to the agent, which lets you get a reference to the current transaction
         IAgent agent = NewRelic.Api.Agent.NewRelic.GetAgent();
         ITransaction currentTransaction = agent.CurrentTransaction;
-        currentTransaction.AcceptDistributedTraceHeaders(httpContext, Getter, TransportType.HTTP);
-        IEnumerable<string> Getter(HttpContext carrier, string key) 
-        { 
-            string value = carrier.Request.Headers[key];   
-            return value == null ? null : new string[] { value }; 
-        }
+
+        // In this example, we are using Azure Service Bus.  The `ServiceBusMessage` type has an `ApplicationProperties` property for custom key/value pairs.
+
+        // Create the outbound message
+        ServiceBusMessage message = new ("Hello, world!");
+
+        // Define the setter Action.  The `ApplicationProperties` dictionary is the trace data carrier.
+        var setter = new Action<ServiceBusMessage, string, string>((carrier, key, value) => { carrier.ApplicationProperties?.Set(key, value); });
+
+        // Call the API to add the distributed tracing data to the message
+        currentTransaction.InsertDistributedTraceHeaders(message, setter);
+
+        // Send the message
         ```
       </Collapser>
 
       <Collapser
-        id=""
-        title="InsertDistributedTraceHeaders"
+        id="AcceptDistributedTraceHeaders"
+        title="AcceptDistributedTraceHeaders"
       >
         ### Syntax
 
         ```cs
-        void InsertDistributedTraceHeaders(carrier, setter)
+        void AcceptDistributedTraceHeaders(carrier, getter, transportType)
         ```
 
-        Implement distributed tracing.
+        Accepts [distributed tracing](/docs/apm/agents/net-agent/configuration/distributed-tracing-net-agent/#manual-instrumentation) data from an inbound message from another instrumented service.
```

Comment:
Either this should be "data from an inbound message to another instrumented service" or "data from another instrumented service's inbound message."